src/ProductBundle/Entity/BaseProduct.php

```diff
@@ -845,41 +848,6 @@ public function isPriceIncludingVat()
     }
 
     /**
-     * source : http://snipplr.com/view/22741/slugify-a-string-in-php/.
-     *
-     * @static
-     *
-     * @param  $text
-     *
-     * @return mixed|string
-     */
-    public static function slugify($text)
```

Comment:
I can keep `setSlug` as it is now:
```php
$slug = Slugify::create()->slugify(trim($slug));

$this->slug = !empty($slug) ? $slug : 'n-a';
```
And keep changed `BaseProduct::slugify` with adding `@deprecated` to it